scripts/tests/test-addon-blueprint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# -e - fails the script if any command errors
+#
+set -e
+
+npm install -g ember
```

Comment:
This seems wrong (should be ember-cli), but you can avoid doing two things via:

    npx ember-cli addon my-addon -b $ADDON_PATH

Reply:
doh. of course it passes on my system, cause I already have e-c installed. thanks